services-js/commissions-search/src/FacetList.css

```diff
@@ -0,0 +1,39 @@
+
+
+input[type="radio"].cb-f:before{
+    content:"";
+    display:block;
+    height:35px;
+    width:35px;
+    background-color:#fff;
+    border:3px solid #091f2f;
+    box-sizing:border-box;
+    position:absolute;
+    top:0;
+    left:0
+}
+
+@media screen and (min-width: 768px) {
+    input[type="radio"].cb-f:before {
+        height: 42px;
+        width: 42px;
+        border-radius: 50%;
+    }
+}
+
+input[type="radio"].cb-f:checked:before{
+    background:#fff url(./icons/check.svg) no-repeat 50%
```

Comment:
Refer to the absolute `patterns.boston.gov` URL to avoid this potentially being relative to whatever page the JS is run on.